airbyte-ci/connectors/pipelines/README.md

```diff
@@ -85,7 +85,7 @@ If you are developing on pipelines, we recommend installing airbyte-ci with poet
 ```bash
 cd airbyte-ci/connectors/pipelines/
 poetry install
-poetry shell
+poetry env activate
```

Comment:
It seems like this command is not available in poetry 2.X but I haven't test if the alternative mentioned [here](https://github.com/python-poetry/poetry/blob/main/CHANGELOG.md?plain=1#L111) actually works